be/src/olap/olap_server.cpp

```diff
@@ -551,4 +537,51 @@ void StorageEngine::_pop_tablet_from_submitted_compaction(TabletSharedPtr tablet
     }
 }
 
+Status StorageEngine::_submit_compaction_task(TabletSharedPtr tablet, CompactionType compaction_type) {
+    bool already_exist = _push_tablet_into_submitted_compaction(tablet, compaction_type);
+    if (already_exist) {
+        return Status::InternalError(strings::Substitute(
```

Comment:
Is `Status::AlreadyExist` more appropriate?

Reply:
> Is `Status::AlreadyExist` more appropriate?

OK, thank you.